php/libraries/TimePoint.class.inc

```diff
@@ -939,4 +939,5 @@ class TimePoint
 
         return $candidate->getCandidateDoB();
     }
-}
+} // end class
```

Comment:
not sure why this comment was added

Reply:
(just a comment, not a blocker.. I'll merge once Travis passes..)

Reply:
I think a rebase error caused it to be added back